integration-tests/test.sh

```diff
@@ -43,11 +43,17 @@ trap "rv=\$?; docker rm -vf $id; exit \$rv" INT TERM EXIT
 out=$(docker_exec "/goss/$os/goss-linux-$arch" --vars "/goss/vars.yaml" -g "/goss/$os/goss.yaml" validate)
 echo "$out"
 
-if [[ $os == "arch" ]]; then
-  egrep -q 'Count: 81, Failed: 0, Skipped: 3' <<<"$out"
-else
-  egrep -q 'Count: 97, Failed: 0, Skipped: 5' <<<"$out"
-fi
+case $os in
+  "arch")
+    egrep -q 'Count: 83, Failed: 0, Skipped: 3' <<<"$out"
+    ;;
+  "wheezy")
```

Comment:
I would prefer to keep wheezy in sync with * if possible. It's already annoying that arch count is different than the rest.

Reply:
The issue was that `wheezy` has a crazy version of `bash` installed that didn't parse correctly. Is that acceptable or should I get another test for `wheezy`?

Reply:
What happens when you run the code against wheezy?

Also, to be 100% clear.. I love this PR, I don't view it as a solution to package versioning.. just useful for working with things that use semver.

See my comment here for why I don't think semver is sufficient:
https://github.com/aelsabbahy/goss/issues/435#issuecomment-486013785

Reply:
Continuing on https://github.com/aelsabbahy/goss/pull/508#issuecomment-561477078

Reply:
The `wheezy` differences were removed.